### ISSUE TITLE
When does the official support yolov3 and export models in onnx format?

### ISSUE THREAD

**Issue by cuixing158** [mathworks-robotics/deep-learning-for-object-detection-yolov2#1]
Yolov3 is a very common algorithm, but Matlab does not support it. Of course, it is not very flexible. It does not support the definition of the three types of loss functions of Yolov3, and the object detection algorithms do not support exporting as onnx models. There are only a few official ones. Simple classification model supports Onnx, And didn't answer any questions on matlab answers?

**Comment by connelld92** [comment]
Hi there, I am adding @davidwillingham who works with the Deep Learning Toolbox who will be able to give you a status update on the YOLOv3 support from MATLAB.

**Comment by davidwillingham** [comment]
@cuixing158 R2020a will include an example for Yolov3

**Comment by cuixing158** [comment]
I reproduced the original version of yolov3 / yolov4, but encountered the following problems: 
1. The automatic differential derivation architecture is not fast (such as GIOU); 
2. The network layer is not easy to control freely, and the location of the retrieval layer Very inefficient; 
3. Network inference is slower than the original framework (even with GPU)
yolov3-yolov4-matlab respository here: https://github.com/cuixing158/yolov3-yolov4-matlab

**Comment by davidwillingham** [comment]
Hi, are you refering to the performance of this example:
https://www.mathworks.com/help/vision/examples/object-detection-using-yolo-v3-deep-learning.html

**Comment by cuixing158** [comment]
Yes, I refer to the official yolov3 column, but the performance of the loss function is greatly reduced by a slight change！The reverse error propagation efficiency is very low, and it is very slow to replace mse with GIOU or others! I hope you can really review the reason! thanks!

**Comment by davidwillingham** [comment]
Hi,

Following up on your questions:

1.	We are aware of this and we are working on improving the dlnetwork framework. 
2.	Can you provide more details on this? 
3.	Thanks your for sharing this repo with us, we will check this against ours.
